demo/api/schema.gql

```diff
@@ -460,6 +460,7 @@ type BrevoConfig implements DocumentInterface {
   updatedAt: DateTime!
   senderMail: String!
   senderName: String!
+  folderId: Int
```

Comment:
This is not nullable right? Why is this not marked with "!" ? 🤔 did the schema not update?

Reply:
b82acde06742ba4bf10ce7aa05202c5a4db90ab6